utils/open_model_zoo/Transportation/semantic-segmentation-adas/interp.py

```diff
@@ -0,0 +1,27 @@
+import numpy as np
+from skimage.measure import approximate_polygon, find_contours
+
+import cv2
+
+
+for frame_results in detections:
+    frame_height = frame_results['frame_height']
+    frame_width = frame_results['frame_width']
+    frame_number = frame_results['frame_id']
+    detection = frame_results['detections']
+    detection = detection[0, 0, :, :]
+    width, height = detection.shape
+
+    for i in range(20):
```

Comment:
@benhoff , does 0 corresponds to background? Should we add it into label_map? Should we have here `range(21)`?

Reply:
Off by one strikes! Good catch.